Android/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(Cuberite)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
+	CMAKE_ARGS -DANDROID_CROSSCOMPILE=TRUE
+	BUILD_COMMAND
+		${CMAKE_COMMAND} --build <BINARY_DIR> --config Release
+	INSTALL_COMMAND ""
+)
+externalproject_add(tolua_native
+	SOURCE_DIR ../lib/tolua++
+	CMAKE_GENERATOR "Visual Studio 14 2015"
+	INSTALL_DIR ${CMAKE_SOURCE_DIR}/../src/Bindings
+	BUILD_COMMAND
+		${CMAKE_COMMAND} --build <BINARY_DIR> --config Release
+	INSTALL_COMMAND
+		${CMAKE_COMMAND} -E copy "${CMAKE_SOURCE_DIR}/tolua_native-prefix/src/tolua_native-build/Release/tolua.exe" <INSTALL_DIR>
+)
+
+# Tell libevent that Android has pthread (not auto detected due to NDK implementation problems)
+# Tell Lua to not use unimplemented Android locale features
+add_definitions(-Dgetlocaledecpoint\(\)="'.'")
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+set(CROSSCOMPILE 1)
+
+add_library(AndroidNative jni/app-android.cpp)
+add_dependencies(tolua_native lua_native)
+add_dependencies(AndroidNative tolua_native)
```

Comment:
I thought it was more like tolua_native depended on AndroidNative. My intention was to get tolua_native and lua_native to actually build.